dashboard/modules/healthz/healthz_agent.py

```diff
@@ -27,16 +27,20 @@ async def health_check(self, req: Request) -> Response:
             alive = await self._health_checker.check_local_raylet_liveness()
             if alive is False:
                 return Response(status=503, text="Local Raylet failed")
-        except grpc.RpcError as e:
+        except ray.exceptions.RpcError as e:
             # We only consider the error other than GCS unreachable as raylet failure
             # to avoid false positive.
             # In case of GCS failed, Raylet will crash eventually if GCS is not back
             # within a given time and the check will fail since agent can't live
             # without a local raylet.
-            if e.code() not in (
-                grpc.StatusCode.UNAVAILABLE,
-                grpc.StatusCode.UNKNOWN,
-                grpc.StatusCode.DEADLINE_EXCEEDED,
+            # Note: we want to cehck the rpc error code, but we can't import grpc since
+            # this is a minimal module. I hardcode these error codes because grpc status
+            # codes are relatively stable. Here is the reference link:
+            # https://grpc.github.io/grpc/core/md_doc_statuscodes.html
+            if e.rpc_code not in (
+                14,  # grpc.StatusCode.UNAVAILABLE,
```

Comment:
We already have wrappers for some of these and it is easy to add them for the ones we don't have yet: https://github.com/ray-project/ray/pull/35705 :)

Reply:
thanks for the pointer!